src/config/dark-sites.config

```diff
@@ -61,6 +61,7 @@ benleggiero.me
 bequiet.com
 bestblackhatforum.com
 beta.destinyitemmanager.com
+betterdiscordlibrary.com
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/109308569-1b83f080-783a-11eb-8ccd-18cb128956bf.png)
not dark

Reply:
This is so weird again, I think this depends on the Windows theme preferences.
Do you have any tips that help me determine if it was a dark site by default or not?

Reply:
Personally I use Firefox + Private Windows + `privacy.resistFingerprinting`(to force white-theme-preference) to check sites.

Reply:
Okay thanks!